src/overlays/actors/ovl_Bg_Spot18_Obj/z_bg_spot18_obj.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_bg_spot18_obj.c
+ * Overlay: Bg_Spot18_Obj
+ * Description:
```

Comment:
This actor is the statue in Daruina's Room. Could you please add a description that mentions this?

Reply:
we generally have been allowing people to submit actors with blank descriptions if they dont feel comfortable naming it based off the code. even though that is 100% right, i dont think we should have people doing it at the time of decomp if they dont want to. it can be taken care of at a documentation step imo

Reply:
I agree that they should be able to be left blank, but since Jack gave me the description here I went ahead and added it. :)

Reply:
I just feel a brief description, something that at least explains what the actor is, wouldn't hurt to have early on. A quick lookup from https://wiki.cloudmodding.com/oot/Actor_List_(Variables) reveals what the actor is, and I think the descriptions from this page could potentially be copied over.

Reply:
we've already seen multiple cases where the descriptions here are wrong. for example it calls en_it dampes minigame collectibles, but its actually the digging game hitboxes.
it should really be given a proper name from what is seen from the code itself, not existing documentation